spec/api/channels_spec.cr

```diff
@@ -60,6 +60,30 @@ describe LavinMQ::HTTP::ChannelsController do
         end
       end
     end
+
+    it "should return get_no_ack count 1" do
+      with_http_server do |http, s|
+        with_channel(s) do |c|
```

Comment:
Very minor, but we pretty much always use `ch` in all other specs, so I think it's easier if we keep to that tradition 🙂